src/client/scripts/esm/game/chess/selection.js

```diff
@@ -116,27 +139,47 @@ function update() {
 		if (promoteTo) makePromotionMove();
 		return;
 	}
+	if (perspective.isLookingUp() && draggingPiece) return draganimation.hideHeldPiece(); //Don't render the draggedPiece if we are looking at the sky.
```

Comment:
I imagine just a return statement here would do. And in the render method, replace the `hidden` variable with a check for if we're looking up, and if we are, don't regen the model or render.